### ISSUE TITLE
IDE inserts import for `Boolean` when `True` is selected from the dropdown

### ISSUE THREAD

**Issue by 4e6** [enso-org/enso#6816]
### What type of issue is this?

Permanent – Occurring repeatably

### What issue are you facing?

IDE inserts a new import for `Boolean` when `True` is selected from the dropdown, even though the `Boolean` type is reexported from `Standard.Base.Main`.

```
from Standard.Base.Data.Boolean import Boolean
```

This does not happen when `True` is selected from the Component Browser.

### Expected behaviour

IDE does not insert additional imports when `True` is selected from the dropdown.

### How we can reproduce it?

- Create `Statistic.Skew` node
- Select `True` from the dropdown
- See the new `from Standard.Base.Data.Boolean import Boolean` import

### Screenshots or screencasts

![2023-05-23-002251_992x632_scrot](https://github.com/enso-org/enso/assets/357683/86c984bb-0ba2-4ec0-bc70-3b12d8335eb5)

### Logs

The engine sends suggestion updates with the proper `reexport` field for `Boolean` type and constructors:
```json
{"type":"Add","id":1615,"suggestion":{"type":"type","module":"Standard.Base.Data.Boolean","name":"Boolean","params":[],"parentType":"Standard.Base.Any.Any","documentation":" A type with only two possible values.\n\nThe boolean type represents the two truth values of boolean logic. It is\nprimarily used for control-flow.","reexport":"Standard.Base.Main"}}

{"type":"Add","id":1616,"suggestion":{"type":"constructor","module":"Standard.Base.Data.Boolean","name":"True","arguments":[],"returnType":"Standard.Base.Data.Boolean.Boolean","reexport":"Standard.Base.Main"}}

{"type":"Add","id":1617,"suggestion":{"type":"constructor","module":"Standard.Base.Data.Boolean","name":"False","arguments":[],"returnType":"Standard.Base.Data.Boolean.Boolean","reexport":"Standard.Base.Main"}}
```

[project-manager.log.zip](https://github.com/enso-org/enso/files/11537224/project-manager.log.zip)

### Enso Version

b353b35f5

### Browser or standalone distribution

Chromium

### Browser Version or standalone distribution

113.0.5672.126

### Operating System

Linux

**Comment by Frizi** [comment]
> This does not happen when True is selected from the Component Browser.

I don't experience the same thing. For me, selecting `true` does introduce an import.

https://github.com/enso-org/enso/assets/919491/9cba8b40-5b1f-49d1-994a-a2aee92abded

**Comment by JaroslavTulach** [comment]
There was a problem with re-exports, but that one got fixed in #6778 - assuming the fix is correct, it is now up to the IDE to adjust to the [updates with reexport](https://github.com/enso-org/enso/pull/6778/files#diff-182e5d2fb4dca39f401ee856c8d94ee7f3f994c5e9268063e13f5c1b24b89eacL521). 

Thanks @Frizi testing. I am not sure what to conclude from your video (except something is broken) - the reexport information shall be there since #6778 - if there is some work needed on the `--compiler` side, please remove the `--gui` label.

**Comment by farmaazon** [comment]
I reproduced the issue exactly as in the description, i.e. the Component Browser does not insert import, but drop-down does.

@Frizi I think you have an older engine in your project. Make sure it's never than nightly 2022-05-18.
